libs/types-common/src/Data/Misc.hs

```diff
@@ -165,9 +177,13 @@ instance ToJSON Location where
 
 instance FromJSON Location where
   parseJSON = withObject "Location" $ \o ->
-    location <$> (Latitude <$> o .: "lat")
+    location
+      <$> (Latitude <$> o .: "lat")
       <*> (Longitude <$> o .: "lon")
 
+instance Arbitrary Location where
+  arbitrary = Location <$> arbitrary <*> arbitrary
```

Comment:
It can be, but the generic instance is currently defined in `wire-api`, which is downstream. Didn't seem very high priority to me, but we can definitely move the `GenericUniform` newtype + instance to `types-common` instead.

Reply:
I decided not to move it for now, since the benefit for these one or two instances is very small (and I don't want to touch most of types-common either). Also, I discovered that it would lead to conflict in another branch I have on top of this one.